recurly/notifications/notifications.go

```diff
@@ -0,0 +1,78 @@
+package notifications
+
+import (
+	"encoding/xml"
+	"io"
+	"io/ioutil"
+
+	"github.com/blacklightcms/go-recurly/recurly"
+)
+
+// Webhook notification constants.
+const (
+	SuccessfulPayment = "successful_payment_notification"
+	FailedPayment     = "failed_payment_notification"
+	PastDueInvoice    = "past_due_invoice_notification"
+)
+
+type notificationName struct {
+	XMLName xml.Name
+}
+
+type (
+	// SuccessfulPaymentNotification is sent when a payment is successful.
+	SuccessfulPaymentNotification struct {
+		Account     recurly.Account
+		Transaction recurly.Transaction
+	}
+
+	// FailedPaymentNotification is sent when a payment fails.
+	FailedPaymentNotification struct {
+		Account     recurly.Account
+		Transaction recurly.Transaction
+	}
+
+	// PastDueInvoiceNotification is sent when an invoice is past due.
+	PastDueInvoiceNotification struct {
+		Account recurly.Account
+		Invoice recurly.Invoice
+	}
+)
+
+// TODO respond to webhook with a 200 code within 5 minutes or Recurly will resend it.
```

Comment:
Yes, I just wanted to ensure that somewhere it is noted that webhooks need to receive a response from our implementation.